orchestration/dagster_orchestration/hca_manage/dataset.py

```diff
@@ -184,6 +187,12 @@ def create_dataset(
             "region": region,
             "cloudPlatform": "gcp"
         }
+
+        if env == 'prod':
```

Comment:
not important at all, but consider using double quotes here to be consistent with the rest of the strings in our format